server/src/test/java/org/elasticsearch/search/SearchRequestTests.java

```diff
@@ -164,4 +181,33 @@ private static SearchRequest copyRequest(SearchRequest searchRequest) throws IOE
         }
         return result;
     }
+
+    private static class NoOpRescorerBuilder extends RescorerBuilder<NoOpRescorerBuilder> {
```

Comment:
I had not looked closely now I see why you were using mockito before :) I don't mind if you prefer to go back to it, I am happy either way